code/datums/abilities/vampire.dm

```diff
@@ -239,6 +241,25 @@
 				owner.full_heal()
 			else
 				changeling_super_heal_step(healed = owner, mult = mult*2, changer = 0)
+		// alerts thralls when dead and not in coffin
+		else if (isdead(owner) && !death_triggered)
+			death_triggered = 1
+			warn_thralls()
+		if (!isdead(owner) && death_triggered)
+			death_triggered = 0
+		// alert thralls when in crit
+		if (owner.health < 0 && !crit_triggered)
```

Comment:
we sadly don't have a mob crit signal...at least not yet. 

Instead of checking health on life tick look into `/mob/proc/updatehealth()`